.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -235,20 +244,20 @@ main() {
   ENCODED_API_SERVER_URL=$(echo "$API_SERVER_URL" | base64)
   ENCODED_CLUSTER_NAME=$(echo "my-cluster" | base64)
 
-  configure_namespace
+  configure_namespace ${NAME_SPACE}
   oc project ${NAME_SPACE}
   installPipelinesOperator $DIR
   install_helm
   uninstall_helmchart
 
   cd $DIR
-  apply_yaml_files $DIR
+  apply_yaml_files $DIR "$NAME_SPACE"
   add_helm_repos
 
   echo "Tag name : ${TAG_NAME}"
-  helm upgrade -i ${RELEASE_NAME} -n ${NAME_SPACE} ${HELM_REPO_NAME}/${HELM_IMAGE_NAME} --version ${CHART_VERSION} -f $DIR/value_files/${HELM_CHART_VALUE_FILE_NAME} --set global.clusterRouterBase=${K8S_CLUSTER_ROUTER_BASE} --set upstream.backstage.image.tag=${TAG_NAME}
+  helm upgrade -i "${NAME_SPACE}" -n ${NAME_SPACE} ${HELM_REPO_NAME}/${HELM_IMAGE_NAME} --version ${CHART_VERSION} -f $DIR/value_files/${HELM_CHART_VALUE_FILE_NAME} --set global.clusterRouterBase=${K8S_CLUSTER_ROUTER_BASE} --set upstream.backstage.image.tag=${TAG_NAME}
```

Comment:
Why has been `${RELEASE_NAME}` removed and replaced with `"${NAME_SPACE}"`?
The current command appears incorrect; we need to include the release name.

Reply:
Even if  ${RELEASE_NAME} and ${NAME_SPACE} have the same value, it would be clearer if we use separate property names. A code reviewer would suspect it as a typo.